src/config/keys.ts

```diff
@@ -54,7 +54,20 @@ export const buildManyItemMembershipsKey = (ids?: UUID[]) => [
 export const buildItemLoginKey = (id?: UUID) => [ITEMS_KEY, id, 'login'];
 export const TAGS_KEY = 'tags';
 export const ITEM_TAGS_KEY = 'itemTags';
+export const itemTagsKeys = {
+  all: [ITEMS_KEY, ITEM_TAGS_KEY] as const,
+  many: () => [...itemTagsKeys.all, 'many'] as const,
+  single: () => [...itemTagsKeys.all, 'single'] as const,
```

Comment:
Not sure if these 3 helper prefix factories should be part of the public API, consumers probably don't want to use these. Consider using a class with private methods, or non-exported functions.

```suggestion
```

Reply:
I see, I didn't know React Query supported key prefix access